state/event_table.go

```diff
@@ -130,7 +130,7 @@ func NewEventTable(db *sqlx.DB) *EventTable {
 	-- index for querying membership deltas in particular rooms
 	CREATE INDEX IF NOT EXISTS syncv3_events_type_room_nid_idx ON syncv3_events(event_type, room_id, event_nid);
 	-- index for querying events in a given room
-	CREATE INDEX IF NOT EXISTS syncv3_nid_room_state_idx ON syncv3_events(room_id, event_nid, is_state);
+	CREATE INDEX IF NOT EXISTS syncv3_nid_room_state_idx ON syncv3_events(room_id, is_state, event_nid);
```

Comment:
```suggestion
	CREATE INDEX IF NOT EXISTS syncv3_events_nid_room_state_idx ON syncv3_events(room_id, is_state, event_nid);
```

Different name so we can upgrade cleanly.